distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Base_Generator.enso

```diff
@@ -138,7 +138,7 @@ make_function name =
 
    Arguments:
    - sql_code: The SQL code for the constant.
-make_constant : Text -> Vector Builder -> Builder
+make_constant : Text -> (Vector Builder -> Builder)
```

Comment:
Is (->) left-associative, or is this for clarity?

Reply:
this function returns a function which takes a `Vector Builder` and returns a `Builder`.